modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -7,7 +7,6 @@ module ClaimsApi
   module V1
     class ClaimsController < ApplicationController
       before_action { permit_scopes %w[claim.read] }
-      before_action :verify_power_of_attorney
```

Comment:
not sure I like this flow as much as calling verify before, leads to the keyword arg that calls a function that calls this...idk it's a weird flow

Reply:
Hey @ATeal I did this because we no longer want to validate on any request, only when certain headers are passed/not passed . I have usually only used conditional filters on certain actions. We could make the case we could do it based on the use_veteran_headers method. The main reason I removed it is that we now have a flow that does not require the verify poa stuff to be called

Reply:
Maybe then the call is to rename the `use_headers_for_veteran_info?` to something like 'vso_request?` or `poa_request`? and `before_action :verify_power_of_attorney, if: :poa_request?` 

Maybe I'm misunderstanding the flow of the code, but it's pretty close to an infinite loop lol

Reply:
I like that idea. Just updated it